evap/staff/importers/json.py

```diff
@@ -0,0 +1,204 @@
+from datetime import datetime, timedelta
+from typing import TypedDict
+
+from django.db import transaction
+
+from evap.evaluation.models import Contribution, Course, CourseType, Degree, Evaluation, Semester, UserProfile
+from evap.evaluation.tools import clean_email
+
+
+class ImportStudent(TypedDict):
+    gguid: str
+    email: str
+    name: str
+    christianname: str
+
+
+class ImportLecturer(TypedDict):
+    gguid: str
+    email: str
+    name: str
+    christianname: str
+    titlefront: str
+
+
+class ImportCourse(TypedDict):
+    cprid: str
+    scale: str
+
+
+class ImportRelated(TypedDict):
+    gguid: str
+
+
+class ImportAppointment(TypedDict):
+    begin: str
+    end: str
+
+
+class ImportEvent(TypedDict):
+    gguid: str
+    lvnr: int
+    title: str
+    title_en: str
+    type: str
+    isexam: bool
+    courses: list[ImportCourse]
+    relatedevents: ImportRelated
+    appointments: list[ImportAppointment]
+    lecturers: list[ImportRelated]
+    students: list[ImportRelated]
+
+
+class ImportDict(TypedDict):
+    students: list[ImportStudent]
+    lecturers: list[ImportLecturer]
+    events: list[ImportEvent]
+
+
+class JSONImporter:
+    DATETIME_FORMAT = "%d.%m.%Y %H:%M"
+
+    def __init__(self, semester: Semester):
```

Comment:
Can you annotate all functions/methods that return `None` as `-> None`? Otherwise, PEP-484/Mypy default to `Any`.